test/integration/content-explorer/ContentExplorer.test.js

```diff
@@ -0,0 +1,37 @@
+// <reference types="Cypress" />
+
+describe('ContentExplorer', () => {
```

Comment:
fwiw, this seems like it is something other than a unit test, and I'd pictured that those would end up being ContentExplorer.int.test.js (or whatever the test type is).  Not saying change it, just saying: maybe we do need some other kind of category here?

Reply:
Cypress is an end-to-end testing library meant to simulate user interactions via a browser. It's not really intended to be used for unit tests. I originally had the files with the `.spec.js` extension, which delineates them against regular unit tests.

Reply:
the folder structure does that already

Reply:
That's true, as well, @j3tan. In that case, it doesn't seem like the `.int.test.js` change is necessary, either.

Reply:
I renamed the test file to `ContentExplorer.e2e.test.js` to identify it as an end-to-end test.